package.json

```
@@ -8,7 +8,7 @@
     "lint": "npx eslint test/ index.js"
   },
   "engines": {
-    "node": "^10.17.0"
+    "node": ">=10.17.0"
```

Old versions of Node 11 do not support this as well, so instead we should change it to the following:

```suggestion
    "node": "^10.17.0 || >= 11.14.0"
```